Mytinerary/View Controllers/DailyCalendarViewController.m

```diff
@@ -11,19 +11,26 @@
  Brainstorming:
     - Probably going to have increments of 15(?) minutes that are cells
     - Use gesture recognizer to detect tap, which adds UIView to the screen?
-        - But can I add coordinates to the event to make sure it spans the timeframe?
- 
  
     Maybe NSCalendar could be useful?
 
  
  Questions:
     - Should formatter be defined as a property of the Daily View Controller?
+    - Possible to add addEventWith() to scheduledEventView.m file?? outsourcing?
+    - Currently iterating through 'events' array of pointers in itinerary, using the ID to fetch said
+            event from parse. More efficient way?
```

Comment:
there must be a way to fetch multiple IDs in 1 fetch with parse... 🤔

Reply:
Maybe this will help:

https://stackoverflow.com/a/31432281

Reply:
Thats super helpful!